plugins/modules/template_workflow_manager.py

```diff
@@ -1231,7 +1248,7 @@ def get_device_types(self, device_types):
             if product_family is not None:
                 deviceTypes[i].update({"productFamily": product_family})
             else:
-                self.msg = "product_family is mandatory for deviceTypes"
+                self.msg = "product_family is required for deviceTypes"
```

Comment:
device_types?

`"The parameter 'product_family' is required for 'device_types' but not provided"`